payment/controllers.go

```diff
@@ -0,0 +1,292 @@
+package payment
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/asaskevich/govalidator"
+	"github.com/brave-intl/bat-go/middleware"
+	"github.com/brave-intl/bat-go/utils/handlers"
+	"github.com/brave-intl/bat-go/utils/requestutils"
+	"github.com/go-chi/chi"
+	uuid "github.com/satori/go.uuid"
+)
+
+// Router for order endpoints
+func Router(service *Service) chi.Router {
+	r := chi.NewRouter()
+	r.Method("POST", "/", middleware.InstrumentHandler("CreateOrder", CreateOrder(service)))
+	r.Method("GET", "/{id}", middleware.InstrumentHandler("GetOrder", GetOrder(service)))
```

Comment:
let's make this consistent and use `orderID` to keep in line with casing for other handlers

Reply:
https://github.com/brave-intl/bat-go/pull/226/commits/df9089b28f7cb6dcad39264edbd127e49dece803